torch/_torch_docs.py

```diff
@@ -4409,6 +4409,10 @@ def parse_kwargs(desc):
 If :attr:`some` is ``True`` (default), the returned `U` and `V` matrices will
 contain only :math:`min(n, m)` orthonormal columns.
 
+.. note:: The implementation of SVD on CPU uses the LAPACK routine `?gesdd` (a divide-and-conquer
```

Comment:
Oh so we are running gesdd in both paths. Given this case, could you rename the function name from `gesvd` to `gesdd`?

Reply:
I was going to ask you about that: shall I proceed to rename all `gesvd` functions to `gesdd` in ATen? I think it should be a simple codemod.

Reply:
Yes please! I think only TH, THC, and a Declaraions.cwrap change is needed, so it's not too bad. Thanks!

Reply:
@SsnL done!!